### ISSUE TITLE
Add list with important information

### ISSUE THREAD

**Issue by TonyToscana** [GuilleAmutio/sos-valencia#3]
Add list to the site with information about donation, volunteers, etc around the country

**Comment by GuilleAmutio** [comment]
Yes, with address, IBANs, needs, contacts. That could be another view, "Puntos de atenciòn" o "Informacion importante", in the navbar
